DS13/code/modules/overmap/combat.dm

```diff
@@ -0,0 +1,507 @@
+/mob/living/carbon/canMobMousedown(atom/object, location, params)
+	. = ..()
```

Comment:
if you're not even returning ..(), there's no point in doing .=..(), just do ..()